src/views/studio/StudioLayout.tsx

```diff
@@ -140,6 +147,12 @@ const StudioLayout = () => {
   )
 }
 
+const fullWidthLayout = css`
+  :root {
+    --sidenav-collapsed-width: 0;
```

Comment:
Are you sure this is a good idea to change this for the full studio layout? Some positioning depends on it (like snackbars). If we just set it to 0, some styling will likely break when the sidenav is visible